### PR TITLE
Added import record procedure

### PR DESCRIPTION
**Pull request by aabramos**
Customers are not aware that, after converting an Entity to a Static Entity, they can also import the records from the database as static records.

### CODE REVIEW

File: src/develop/data/modeling/entity.md [hunk 1]
```diff
@@ -43,3 +43,13 @@ When you create a new entity attribute, the platform automatically manages the u
...
 
 When you delete an entity or an entity attribute, the platform is permissive and lets you do it whether it is being used or not, but you must fix the elements where it is being used. In the database, the entity or entity attribute is not deleted by the platform.
+
+## Convert Entity to Static Entity
+You can convert existing Entities to Static Entities. To convert an Entity to Static Entity, right-click on the Entity, navigate to the Advanced help menu and then select Convert to Static Entity.
+
+After converting an Entity to a Static Entity:
+
+* The database records from the won't be available through database queries, via Aggregate or SQL Query
+* The records from the database can be imported as static records. To import the database records, right-click on the Entity, access to the Edit Entity menu item and then select "Import from Database"
```
Comment (carlosrijo) [inline src/develop/data/modeling/entity.md:53]: To keep it coherent I would switch:
> right-click on the Entity
to:
> right-click on the Static Entity

File: src/develop/data/modeling/entity.md [hunk 2]
```diff
@@ -43,3 +43,13 @@ When you create a new entity attribute, the platform automatically manages the u
 When you set an entity attribute as mandatory it is automatically validated on the user interface by the platform. However, in the database, mandatory attributes are created allowing null values thus, at the database level there's no validation for mandatory attributes.
 
 When you delete an entity or an entity attribute, the platform is permissive and lets you do it whether it is being used or not, but you must fix the elements where it is being used. In the database, the entity or entity attribute is not deleted by the platform.
+
+## Convert Entity to Static Entity
+You can convert existing Entities to Static Entities. To convert an Entity to Static Entity, right-click on the Entity, navigate to the Advanced help menu and then select Convert to Static Entity.
+
+After converting an Entity to a Static Entity:
+
+* The database records from the won't be available through database queries, via Aggregate or SQL Query
```
[the remark this answers is not in the document]
Comment (carlosrijo) [reply src/develop/data/modeling/entity.md:52]: I don't have historical context on whether this should work or not but I did some testing and I observed Aggregates and SQL Queries keep working after an entity is converted.

**Review by carlosrijo — CHANGES_REQUESTED** [review]

**Comment by zohav** [comment]
Improvements you made have been incorporated in a separate pull request because of a major restructuring of the technical knowledge portfolio. Your pull request is being closed. Thank you for your contribution.

**Comment by carlosrijo** [comment]
> Improvements you made have been incorporated in a separate pull request because of a major restructuring of the technical knowledge portfolio. Your pull request is being closed. Thank you for your contribution.

Mind sharing the link of the pull request please?

**Pull request closed without merging**